### PR TITLE
content review

### PR DESCRIPTION
**Pull request by jbatynski**
Please change "tab" to "click" where applicable.

### CODE REVIEW

**Comment by AnnKruQA** [comment]
I prefer Tab.

File: User Guide Adding a Photo on Facebook by Anna Krukowska.md [hunk 1]
```diff
@@ -184,7 +183,7 @@ Type alternative text to add a meaningful description to your photo for assistiv
 | 3. Create a New Post | Tap "What's on your mind?" | Tap "What's on your mind?" | Tap "What's on your mind?" | Locate "What's on your mind?" box |
 | 4. Add a Photo | Tap camera icon, choose from gallery or take photo | Tap camera icon, choose from gallery or take photo | Tap camera icon, choose from gallery or take photo | Click camera icon, upload photo |
 | 5. Write a Caption | Add a caption | Add a caption | Add a caption | Add a caption |
-| 6. Post Your Photo | Tap "Post" | Tap "Post" | Tap "Post" | Click "Post" |
```
Comment (AnnKruQA) [inline User Guide Adding a Photo on Facebook by Anna Krukowska.md:187]: I prefer Tab.

Comment (jbatynski) [reply User Guide Adding a Photo on Facebook by Anna Krukowska.md:187]: Ok, be my guest ;)

**Review by AnnKruQA — APPROVED** [review]
I prefer "Tab" for mobile.

**Pull request merged**